src/components/API/API.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  ROOT_ENDPOINT,
+  BEERS,
+  PAGE_NUMBER,
+  PER_PAGE,
+  BEER_NAME,
+  SINGLE_BEER,
+} from './data';
+import { IBeerDetails } from '../ItemList/data';
+
+function buildQueryString(
+  searchString: string | null,
+  pageNumber: number,
+  perPage: number
+): string {
+  let queryString: string;
+
+  if (searchString) {
+    queryString =
+      ROOT_ENDPOINT +
+      BEERS +
+      PAGE_NUMBER +
+      pageNumber +
+      PER_PAGE +
+      perPage +
+      BEER_NAME +
+      searchString.trim();
+  } else {
+    queryString =
+      ROOT_ENDPOINT + BEERS + PAGE_NUMBER + pageNumber + PER_PAGE + perPage;
+  }
+
+  return queryString;
+}
+
+async function sendGetQuery(
+  queryString: string
+): Promise<IBeerDetails[] | Error> {
+  let responseJSON: IBeerDetails[] | Error;
+
+  try {
+    const responsePromise = await fetch(queryString, {
+      method: 'GET',
+    });
+    responseJSON = await responsePromise.json();
+  } catch (error) {
+    responseJSON = new Error(
+      'Cannot reach out to server. Please, check your network connection and server availability.'
+    );
+  }
+
+  return responseJSON;
+}
+
+async function queryItems(
+  searchString: string | null,
+  pageNumber: number = 1,
+  perPage: number = 4

```

Comment:
Create please variable for this value

Reply:
Fixed in https://github.com/ylakhtin/rs-react-components/commit/fb873d74a06f67fee2d0e4ab1162cf011fe4b438